plugins/monetdb-adapter/src/main/java/org/polypheny/db/adapter/monetdb/MonetdbPlugin.java

```diff
@@ -16,49 +16,44 @@
 
 package org.polypheny.db.adapter.monetdb;
 
-import com.google.common.collect.ImmutableMap;
-import java.util.Map;
-import org.pf4j.Plugin;
-import org.pf4j.PluginWrapper;
+import org.polypheny.db.adapter.AdapterManager;
 import org.polypheny.db.adapter.monetdb.sources.MonetdbSource;
 import org.polypheny.db.adapter.monetdb.stores.MonetdbStore;
-import org.polypheny.db.catalog.Adapter;
+import org.polypheny.db.plugins.PluginContext;
+import org.polypheny.db.plugins.PolyPlugin;
+import org.polypheny.db.sql.language.SqlDialectRegistry;
 
-public class MonetdbPlugin extends Plugin {
+public class MonetdbPlugin extends PolyPlugin {
 
 
-    public static final String ADAPTER_NAME = "MONETDB";
+    public static final String ADAPTER_NAME = "MonetDB";
+    private long storeId;
+    private long sourcId;
```

Comment:
There is a "e" missing. And why two variables anyway? Isn't one ID field sufficient. Whether it's a store or source is anyway determined differently.

Reply:
These ids concern the registered templates not the concrete adapters, therefore both are needed. But I renamed them to make this clearer. Also fixed typo.